packages/shared/locales/en.json

```
@@ -707,7 +707,8 @@
                     "setAsPrimary": "Set as primary",
                     "unsetAsPrimary": "Unset as primary"
                 }
-            }
+            },
+            "notification": "Choose a main account in Account Management to access the dashboard."
```

When this message appears, there is no main account yet
```suggestion
            "notification": "Choose a pending account to activate it."
```